app/models/service_specification.rb

```diff
@@ -78,11 +78,11 @@ def validate_json_format(attribute)
   def validate_numericality_of_json(attribute)
     return if self[attribute].blank?
 
-    errors.add(attribute, :not_an_unsigned_integer) unless values_numeric?(self[attribute].values)
+    errors.add(attribute, :not_a_positive_currency_amount) unless values_numeric?(self[attribute].values)
```

Comment:
But wouldn't this add it for -5 as well? Or is numeric always positiv?

Reply:
Just bad method name